daprdocs/content/en/reference/api/state_api.md

```diff
@@ -292,6 +292,132 @@ None.
 curl -X "DELETE" http://localhost:3500/v1.0/state/starwars/planet -H "If-Match: xxxxxxx"
 ```
 
+## State query
+
+This endpoint lets you query the state.
+
+{{% alert title="alpha" color="warning" %}}
+This API is in alpha stage.
+{{% /alert %}}
+
+### HTTP Request
+
+```
+POST/PUT http://localhost:<daprPort>/v1.0-alpha/state/<storename>/query
+```
+
+#### URL Parameters
+
+Parameter | Description
+--------- | -----------
+daprPort | the Dapr port
+storename | ```metadata.name``` field in the user configured state store component yaml. Please refer Dapr State Store configuration structure mentioned above.
```

Comment:
```suggestion
storename | ```metadata.name``` field in the user configured state store component yaml. Refer to the Dapr state store configuration structure mentioned above.
```

Reply:
There should be no "pleases" in this document. Could you update all the entries of 
" Please refer Dapr State Store configuration structure mentioned above."
to 
"Refer to the Dapr state store configuration structure mentioned above."